src/main/scala/se/lth/immun/TargetFile.scala

```diff
@@ -32,7 +32,7 @@ class TargetFile {
 	
 	def read(path:String, verbose:Boolean):Seq[Target] = {
 		val targets = read(path)	
-		if (verbose) {
+		if (false && verbose) {
```

Comment:
Another left-over from a debug session?

Reply:
I think this was just because it created too much logging output when searching for thousands of targets. Perhaps we can just print the first 10 or 20 targets for debugging purposes.

Reply:
Sure, cap the printed targets (preferably behind a config).